pkg/db/v1beta1/common/const.go

```diff
@@ -46,4 +46,6 @@ const (
 	DefaultPostgreSQLDatabase = "katib"
 	DefaultPostgreSQLHost     = "katib-postgres"
 	DefaultPostgreSQLPort     = "5432"
+
+	SkipDbMigrationEnvName = "SKIP_DB_MIGRATION"
```

Comment:
Should we name it `SKIP_DB_INITIALIZATION` or we need to be consistent with MLMD ?

Reply:
+1 on `SKIP_DB_INITIALIZATION`.